IRONIC.md

```diff
@@ -1 +1,21 @@
 # Ironic Configuration
+
+This document will guide you through the necessary steps to properly configure
+and setup Ironic in RPC-O.
+
+## Variable Overrides
+
+### Set up ironic endpoints for Swift/Ironic health checks
+
+The ironic agent needs to be on the ironic-tftp network in order to access
+ironic-api and Swift.
+
+The following variables need to be inserted and upated with the correct values
+inside of `` user_osa_variables_overrides ``.
+
+	extra_lb_vip_addresses: Insert the IP addresses here 
```

Comment:
I'd use the same format for your 'IP address' variable. also, the format of `extra_lb_vip_addresses` needs to be fixed, it should be something like this.

    extra_lb_vip_addresses:
      - <IP address>